### DISCUSSION TITLE
Lenovo LOQ works

### DISCUSSION THREAD

**Discussion by Jalopy-Tech** [4JX/L5P-Keyboard-RGB#127] [General]
I have a Lenovo LOQ (16IRH8) [2023]. I added the Lenovo LOQ keyboard to the KNOWN_DEVICE_INFOS list in L5P-Keyboard-RGB/driver/src/lib.rs and it compiles and works fine:
`(0x048d, 0xc983, 0xff89, 0x00cc), // 2023 LOQ`

The udev rule is:
`SUBSYSTEM=="usb", ATTR{idVendor}=="048d", ATTR{idProduct}=="c983", MODE="0666"`

Thanks for this awesome application. Let me know if there is anything I can do.

**Comment by FrenzyyLumina** [comment]
Lenovo LOQ 15IRX9 2024 also works

**Comment by Nikorasu-A** [comment]
Also working in Lenovo LOQ 

LOQ 15IRH8

The rule was: SUBSYSTEM=="usb", ATTR{idVendor}=="048d", ATTR{idProduct}=="c983", MODE="0666"

Also thanks to the creator!
